src/__tests__/base-queries-warn-on-invalid-container.js

```diff
@@ -112,7 +112,7 @@ describe('asynchronous queries throw on invalid container type', () => {
     ['findAllByTestId', findAllByTestId],
   ])('%s', (_queryName, query) => {
     const queryOptions = {}
-    const waitOptions = {timeout: 1}
+    const waitOptions = {timeout: 1, onTimeout: e => e}
```

Comment:
I added `onTimeout: e => e` to several places in the tests because otherwise the snapshots got formatted really weird and would fail due to formatting issues 🤷‍♂️

So I added a special test to handle this case specifically and the rest just do nothing to change the error.